### PR TITLE
[12.0][FIX] agreement_legal

### LINKED ISSUE #501: [12.0][BUG] agreement_legal: error upgrading (stage_type)
It might be #355 that add a stage_type but without updating existing data.

> 2020-04-17 09:04:11,553 7 ERROR ygol-stag-apr14-1034772 odoo.sql_db: bad query: ALTER TABLE "agreement_stage" ALTER COLUMN "stage_type" SET NOT NULL
> ERROR: column "stage_type" contains null values
> ****

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Closes #501

### CODE REVIEW

File: agreement_legal/migrations/12.0.2.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2018 - TODAY, Pavlov Media
```
Comment (max3903) [inline agreement_legal/migrations/12.0.2.0.0/post-migration.py]:
```suggestion
# Copyright (C) 2020 Serpent Consulting Services
```

**Review by max3903 — APPROVED** [review]

**Review by ygol — APPROVED** [review]
LGTM

File: agreement_legal/migrations/12.0.2.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
```
Comment (bodedra) [inline agreement_legal/migrations/12.0.2.0.0/post-migration.py:4]:
```suggestion
from openupgradelib import openupgrade
```

File: agreement_legal/migrations/12.0.2.0.0/post-migration.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+def migrate(env, version):
```
Comment (bodedra) [inline agreement_legal/migrations/12.0.2.0.0/post-migration.py:5]:
```suggestion
@openupgrade.migrate()
def migrate(env, version):
```

**Review by bodedra — APPROVED** [review]
Added suggestion. Is it needed or will it work without it?

File: agreement_legal/migrations/12.0.2.0.0/post-migration.py (hunk 3 above)
Comment (ygol) [inline agreement_legal/migrations/12.0.2.0.0/post-migration.py:5]: please do not add unecessary dependencies.

Comment (bodedra) [reply agreement_legal/migrations/12.0.2.0.0/post-migration.py:5]: Yes. I checked with @max3903 and It is not necessary.

**Comment by bodedra** [comment]
@ygol I have deleted my suggestion.

**Pull request merged**